afula/repositories.cfg

```diff
@@ -1,5 +1,5 @@
 [afula]
-PATH=https://github.com/platform-engineering-org/afula
+URL=https://github.com/platform-engineering-org/afula.git
 
 [akka]
-PATH=https://github.com/platform-engineering-org/akka
+URL=https://github.com/platform-engineering-org/akka.git
```

Comment:
is akka really needed here? What is the purpose of adding it?

Reply:
It is just an example of a renovated repo. The Renovate repo manager would scan this repo on top of others.